x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/remove.ts

```diff
@@ -35,33 +35,19 @@ export const deleteTransforms = async (
   await Promise.all(
     transformIds.map(async (transformId) => {
       // get the index the transform
-      const transformResponse = await esClient.transform.getTransform(
-        { transform_id: transformId },
-        { ignore: [404] }
-      );
+      await esClient.transform.getTransform({ transform_id: transformId }, { ignore: [404] });
```

Comment:
What the point of that call now? as we ignore 404 it seems to check the transform exists

Reply:
Great catch! Removed here [`656dd9d` (#159814)](https://github.com/elastic/kibana/pull/159814/commits/656dd9d9388ff2872173c17477ef43b870097161)